src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState, useMemo } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PersonItem } from '../PersonItem';
+import React from 'react';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const isPeopleEmpty = people.length === 0;
+
+  const preparePeopleWithLinks = (peopleList: Person[]) => {
+    return peopleList.map(person => {
+      const { fatherName, motherName } = person;
+
+      return {
+        ...person,
+        fatherNameLink:
+          peopleList.find(p => p.name === fatherName)?.slug || null,
+        motherNameLink:
+          peopleList.find(p => p.name === motherName)?.slug || null,
+      };
+    });
+  };
```

Comment:
Move this function outside the component, because now it's recreating after every render